web/yo/app/scripts/directives/addtherapy.js

```diff
@@ -77,10 +77,11 @@ angular.module('oncokbApp')
 
                 $scope.loadDrugs = function($query) {
                     $scope.addTherapyError = false;
+                    var lowerCaseQuery = $query.toLowerCase();
+                    getDrugsInOrder(lowerCaseQuery);
                     return drugs.filter(function (drug) {
```

Comment:
And I would rather do a filter then a sort to make it more efficient. Also you don't really need to sign a weight on items you don't want to include.